apps/state/lib/state/shape.ex

```diff
@@ -114,6 +114,7 @@ defmodule State.Shape do
     [trip] =
       trips
       |> sort_by_number_of_trips_with_headsign()
+      |> Enum.sort_by(& &1.alternate_route)
```

Comment:
@paulswartz Does this change make sense? I added this because of [this test](https://github.com/mbta/api/blob/master/apps/state/test/state/shape_test.exs#L388), which was [failing like this (line 627)](https://github.com/mbta/api/actions/runs/3968233926/jobs/6801133770) after the upgrade. I traced it down to this function (or perhaps `sort_by_number_of_trips_with_headsign`) and from what I can tell, without a change like this it is not considering the `alternate_route` property?

Reply:
It looks like somewhere between the last OTP 24 release and OTP 25, the ordering of these structs has changed. I looked through the [diff](https://github.com/erlang/otp/compare/OTP-24.3.4.8...OTP-25.0) and didn't see anything obvious there. 

I think this is a good fix for the issue, since we shouldn't be depending on undefined ordering of the structs anyways.